terraform/aws/modules/compute/main.tf

```diff
@@ -104,29 +104,42 @@ resource "aws_ecs_service" "km_ecs_service" {
     container_port   = 80
   }
   network_configuration {
-    assign_public_ip = true
+    assign_public_ip = false
     subnets          = var.private_subnet
-    security_groups  = [ var.elb_sg ]
+    security_groups  = [var.elb_sg]
   }
   tags = merge(var.default_tags, {
   })
 }
 
 resource "aws_cloudwatch_log_group" "km_log_group" {
   name              = "km_log_group_${var.environment}"
-  retention_in_days = 1
+  retention_in_days = 120
 
   tags = merge(var.default_tags, {
     Name = "km_log_group_${var.environment}"
   })
+
+  kms_key_id = "<kms_key_id>"
 }
```

Comment:
**Accurics remediation**
 
**[LOW]: Ensure detailed monitoring is enabled for AWS EC2 instances**
 
**Vulnerability**
AWS EC2 instances without a detailed monitoring could have an impact on incident response.
 
**Remediation**
In AWS Console - 
 1. Sign in to AWS Console and go to the Amazon EC2 console.
 2. In the navigation pane, select Instances.
 3. Select the instance and select Actions, Monitoring, Manage detailed monitoring.
 4. On the Detailed monitoring detail page, for Detailed monitoring, select the Enable check box.
 5. Select Save.

source: accurics-devenv